relevanceai/api/endpoints/deployables/dashboard.py

```diff
@@ -0,0 +1,138 @@
+from abc import ABC, abstractmethod
```

Comment:
I think move this into a separate folder like: 

```
relevanceai/dashboard/dashboard.py 
```

Reply:
Will do